codemeta.json

```
@@ -0,0 +1,269 @@
+    "license": "https://opensource.org/licenses/BSD-3-Clause",
+    "version": "0.15",
+    "author": [
+        {
+            "@type": "Person",
+            "givenName": "Daniel",
+            "additionalName": "G. A.",
+            "familyName": "Smith",
+            "affiliation": "The Molecular Sciences Software Institute",
+            "@id": "https://orcid.org/0000-0001-8626-0900",
+            "identifier": "https://github.com/dgasmith"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Asem",
+            "familyName": "Alenaizan",
+            "affiliation": "Center for Computational Molecular Science and Technology, Georgia Institute of Technology",
+            "@id": "https://orcid.org/0000-0002-0871-664X",
+            "identifier": "https://github.com/alenaizan"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Doaa",
+            "familyName": "Altarawy",
+            "affiliation": "The Molecular Sciences Software Institute",
+            "@id": "https://orcid.org/XXXX",
+            "identifier": "https://github.com/doaa-altarawy"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Taylor",
+            "additionalName": "A.",
+            "familyName": "Barnes",
+            "affiliation": "The Molecular Sciences Software Institute",
+            "@id": "https://orcid.org/XXXX",
+            "identifier": "https://github.com/taylor-a-barnes"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Lori",
+            "additionalName": "A.",
+            "familyName": "Burns",
+            "affiliation": "Center for Computational Molecular Science and Technology, Georgia Institute of Technology",
+            "@id": "https://orcid.org/0000-0003-2852-5864",
+            "identifier": "https://github.com/loriab"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Nuwan",
+            "familyName": "De Silva",
+            "affiliation": "Department of Physical and Biological Sciences, Western New England University",
+            "@id": "https://orcid.org/XXXX",
+            "identifier": "https://github.com/nuwandesilva"
+        },
+        {
+            "@type": "Person",
+            "givenName": "David",
+            "additionalName": "XXXX.",
+            "familyName": "Dotson",
+            "affiliation": "XXXX OpenFF",
+            "@id": "https://orcid.org/XXXX",
+            "identifier": "https://github.com/dotsdl"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Sebastian",
+            "familyName": "Ehlert",
+            "affiliation": "Mulliken Center for Theoretical Chemistry, Institut f\u0188r Physikalische und Theoretische Chemie, Universit\u0164t Bonn",
+            "@id": "https://orcid.org/0000-0001-7809-771X",
+            "identifier": "https://github.com/awvwgk"
+        },
```

Happy to be listed here.

Unless unicode escape sequences are preferred over actual unicode I would suggest:
```suggestion
        {
            "@type": "Person",
            "givenName": "Sebastian",
            "familyName": "Ehlert",
            "affiliation": "Mulliken Center for Theoretical Chemistry, Institut für Physikalische und Theoretische Chemie, Universität Bonn",
            "@id": "https://orcid.org/0000-0001-7809-771X",
            "identifier": "https://github.com/awvwgk"
        },
```

---

I can't remember why I used unicode escape sequences for the Psi4 one of these. But until some validator or scraper protests, I'm glad to go with the actual characters.